src/common/components/select-repository-list/index.js

```diff
@@ -196,9 +184,9 @@ export class SelectRepositoryListComponent extends Component {
               { selectedRepositories.length } selected
             </strong>
             {' '}
-            (<a href="#" onClick={this.onHelpClick.bind(this)}>
+            (<Button appearance={ 'link' } onClick={this.onHelpClick.bind(this)} aria-label={ this.state.showMissingReposInfo ? 'Click to close the missing / private repos dialog' : 'Click to open the missing / private repos dialog' }>
```

Comment:
Ah, now I understand the confusion. [The text I specced](https://docs.google.com/document/d/1koTwI3cRr2S2vj5zHKKbR7aWxLRfShDVq2gHQ29cDyU/edit#heading=h.1uxwfvh1rvjz) was not “Anything missing”, it was “{total} repos (**Any** missing?)”. In this context, “Any” obviously means “any repos”. But I guess it isn’t obvious in the code, where this part of the string is implemented in a separate file (select-repository-list/index.js) from the rest of the string (selectors/index.js). I’ve submitted #862 to fix the text.

(Ideally, yes, I do think the link would link to an ID, so you could bookmark the explanation, open it in a new tab, etc. But that probably shouldn’t be fixed until after the dialog as a whole becomes a fragment of the “My repos” URL, as specced and as possibly required to fix #400 and/or #829.)